File: lyrics.py
import time

def show_lyrics(lyrics, type_speed, pause):
    for char in lyrics:
        print(char, end='', flush=True)
        time.sleep(type_speed / 1000.0) 
    time.sleep(pause / 1000.0)
    print()



show_lyrics("Cause no one will love you,", 94, 210)
show_lyrics("Like her", 92, 43)
show_lyrics("It's pointless", 95, 55)
show_lyrics("Like tears in the rain", 110, 3010)
show_lyrics("So now that she's gone", 123, 410)
show_lyrics("Embrace all that comes", 134, 113)
show_lyrics("And die with a smile", 124, 213)
show_lyrics("Don't show the world how alone you've become", 174, 33)

File: lyrics.cpp
#include <iostream>
#include <chrono>
#include <thread>

using namespace std;

void showLyrics( const string lyrics, int typeSpeed,int pause){
    for(int i = 0; i < lyrics.length();i++){
        cout<<lyrics[i]<<flush;
        this_thread::sleep_for(chrono::milliseconds(typeSpeed));
    }
    this_thread::sleep_for(chrono::milliseconds(pause));
    cout<<endl;
}


int main()
{
    showLyrics("Cause no one will love you,",94,210);
    showLyrics("Like her",92,43);
    showLyrics("It's pointless",95,55);
    showLyrics("Like tears in the rain",110,3010);
    showLyrics("So now that she's gone",123,410);
    showLyrics("Embrace all that comes",134,113);
    showLyrics("And die with a smile",124,213);
    showLyrics("Don't show the world how alone you've become",174,33);
    

    return 0;
}